html/js/host.js

```diff
@@ -92,6 +92,15 @@ $('document').ready(function() {
             return
         }
 
+        if (cmd == "output_atom") {
```

Comment:
not sure why this is still here.
it's already part of the latest code.
https://github.com/moddevices/mod-ui/blob/master/html/js/host.js#L104